src/main/java/org/opensearch/performanceanalyzer/commons/stats/CommonStats.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.performanceanalyzer.commons.stats;
+
+/* Catalogue class that is to be populated upon PerformanceAnalyzerApp class load
+and to be used by both PA and PA-RCA as well as commons repo.
```

Comment:
Can we update the comment as : 

" Catalog *Operational Metrics* class that is to be populated upon PerformanceAnalyzerApp class load and to be used by both PA and PA-RCA. "

One other question, is this class used at runtime by the commons repo ?